woodstock/voting/views.py

```diff
@@ -44,11 +44,17 @@ def events(request):
 
 @login_required
 def dashboard(request):
+    # Parse blind/event values
+    blind = bool(int(request.GET.get('blind', u'1')))
```

Comment:
Since `blind` is used only on templates with values 0 or 1, I suggest we don't convert it to boolean. It's simpler to pass it through as integer.